app/models/post.rb

```diff
@@ -1,3 +1,15 @@
 class Post < ApplicationRecord
   belongs_to :user
```

Comment:
- Good job done so far, as the previous reviewer mentioned you have not fully implemented the relationships between the post model and the `comment` model and `likes` model. Please note that from the ERD a post `has_many` comments and `has_many` likes. Please fix this 🙏

Reply:
@mckent05 done